config/default/common/config/metadata/stories/land_disturbance/step008.md

```diff
@@ -0,0 +1 @@
+This is the same area of southern California and dates, but with the DIST-ALERT layer overlaid on the HLS imagery. The burned areas that are visible in the imagery are confirmed by DIST-ALERT and show the definite vegetation disturbance in that area.
```

Comment:
I think this reads better as "This is the same area and and dates in southern California..."